.github/workflows/CI.yml

```diff
@@ -21,31 +21,22 @@ jobs:
           - '3.1'
           - '3.0'
           - '2.7'
-          - '2.6'
-          - '2.5'
         os:
           - ubuntu
           - macos
           #- windows Tests are failing with Windows for now.
         gemfile:
           - no_rails
-          - rails_5
-          - rails_6
           - rails_6.1
           - rails_7
         exclude:
           - os: macos
             ruby: head
-          - ruby: head
+        include:
+          - ruby: '2.7'
             gemfile: rails_5
```

Comment:
Looks like 5 is actually EOL as well https://www.fastruby.io/blog/ruby/rails/versions/compatibility-table.html
Should it get dropped too?

Reply:
sure